repos/system_upgrade/el7toel8/libraries/config/tests/test_getenvvars.py

```diff
@@ -32,9 +18,9 @@ def test_get_all_vars(monkeypatch):
     envars = {'LEAPP_DEVEL_SKIP_WIP': '0',
               'LEAPP_DEVEL_SKIP_DIP': '1',
               'LEAPP_DEVEL_SKIP_RIP': '2'}
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars))
+    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars=envars))
     assert api.current_actor().configuration.leapp_env_vars == get_all_envs()
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked())
+    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked)
```

Comment:
beaware that this is not equivalent. in you case, when the tested function affects anything inside, you will not recognize it, as you are setting "class" instead of its instance of it. so in case. keep the brackets there rather in all such cases. imagine, that logger should be part of the class as well. in suchcase, you will not be able to access the results after a "test".